src/core/qobj.jl

```diff
@@ -661,6 +661,138 @@ function normalize!(x::Ket)
     return x
 end
 
+"""
+    get_measurement_probabilities(x::Ket, [target_bodies::Vector{<:Integer},
+        hspace_size_per_body::Integer=2])
+
+Returns a vector listing the measurement probabilities of the `target_bodies` of `Ket` `x`.
+
+The Hilbert space size per body can be specified by providing a value for the
+`hspace_size_per_body` argument. If only `x` is provided, the probabilities are provided for
+all the bodies.
+
+The measurement probabilities are listed from the smallest to the largest computational
+basis state. For instance, for a 2-qubit `Ket`, the probabilities are listed for 00, 01, 10,
+and 11.
+# Examples
+The following example constructs a `Ket`, where the probability of measuring 00 is 50% and
+the probability of measuring 10 is also 50%.
+```jldoctest get_measurement_probabilities
+julia> ψ = 1/sqrt(2)*Ket([1, 0, 1, 0]);
+
+julia> print(ψ)
+4-element Ket:
+0.7071067811865475 + 0.0im
+0.0 + 0.0im
+0.7071067811865475 + 0.0im
+0.0 + 0.0im
+
+julia> get_measurement_probabilities(ψ)
+4-element Vector{Float64}:
+ 0.4999999999999999
+ 0.0
+ 0.4999999999999999
+ 0.0
+
+```
+
+For the same `Ket`, the probability of measuring qubit 2 and finding 0 is 100%.
+```jldoctest get_measurement_probabilities
+julia> target_qubit = [2];
+
+julia> get_measurement_probabilities(ψ, target_qubit)
+2-element Vector{Float64}:
+ 0.9999999999999998
+ 0.0
+
+```
+"""
+function get_measurement_probabilities(x::Ket)
```

Comment:
Specifying the return type would be useful for a user. It is not ideal if a user should run or look at the code to determine the function's return type.

This applies to all `get_measurement_probabilities` functions.